### ISSUE TITLE
[feature quest] dyeable redstone dust

### ISSUE THREAD

**Issue by lublak** [dannydjdk/Tiny-Redstone#6]
Change to tint of the redstone dust, to make it easier to see which lines go where.

**Comment by Fireztonez** [comment]
Yeah, the Dyable redstone dust will be really really usefull question to make more complexe circuit and will add a lot of possibility. I use a lot color redstones path with Super Circuit Maker back in 1.10.2 modpacks.

In addition to the Dyable Redstone, having a **Bundle Cable**, how transfer signal from all color circuit in one, but still keep the signal code, will be really really usefull.

If this can help to understand what I talking about, you can see on this video a good and simple example: https://youtu.be/7w004vCZ1LA?t=1130

**Comment by lublak** [comment]
@Fireztonez My idea was actually just to distinguish different lines of redstone dust.
It does not help to create "more complicated circuits" but only to keep different lines viusell apart.
A bundle cable would then be more of an add-on mod.

**Comment by tobi-blackscale** [comment]
Bundled Cables would be cool, maybe even with compatibility to MoreRed(https://www.curseforge.com/minecraft/mc-mods/more-red)?

**Comment by dannydjdk** [comment]
The new [Tiny Pipes](https://www.curseforge.com/minecraft/mc-mods/tiny-pipes) add-on mod adds bundled redstone cables. This doesn't fit the original request, but I thought I'd mention it here. The idea is that each connection on the Redstone Pipe can be dyed, so outputs will only output signals from inputs with matching colors. This allows up to 16 redstone signals to be carried in one pipe.
